src/sentry/south_migrations/0455_auto__add_field_savedsearch_is_global__chg_field_savedsearch_project__.py

```diff
@@ -0,0 +1,1281 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.utils.db import is_postgres
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+
+        # Changing field 'SavedSearch.project'
+        # Adding field 'SavedSearch.is_global'
+        if is_postgres():
+            db.execute("ALTER TABLE sentry_savedsearch ALTER COLUMN project_id DROP NOT NULL")
+            db.execute("ALTER TABLE sentry_savedsearch ADD COLUMN is_global boolean NULL DEFAULT False;")
```

Comment:
Updated to set the on `ALTER COLUMN` instead, which means it'll just apply to rows going forward.

Reply:
https://www.postgresql.org/docs/9.3/sql-altertable.html under `SET/DROP DEFAULT`. The default is still in place in mysql/sqlite, which I assume is fine for on premise?

Generally I prefer to enforce things at the db when possible, but if you prefer I can just remove it, since Django will handle it anyway